pkg/policy/encode.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/nspcc-dev/neofs-api-go/pkg/netmap"
 )
 
+//Encode parses data of PlacementPolicy to a string.
```

Comment:
Space after `//` ("spaceless" comments are usually used for automatic tools in Go, like `//nolint` or `//go:generate`).

Reply:
Fixed.